README.md

```diff
@@ -1,19 +1,28 @@
-# Class-Profile1A-Website
-Website for the SYDE 2025 1A Class Profile.
+# Class-Profile2A-Website
+Website for the SYDE 2025 2A Class Profile.
 
 ## Table of Contents
-- [SYDE2025-1A-Class-Profile-](#syde2025-1a-class-profile-)
+- [SYDE2025-2A-Class-Profile](#syde2025-1a-class-profile-)
   * [About The Project](#about-the-project)
-    + [Purpose](#purpose)
-    + [Languages, Tools and Libraries](#languages-tools-and-libraries)
+  * [Languages, Tools and Libraries](#languages-tools-and-libraries)
+  * [Build instruction](#build-instruction)
   * [License](#license)
   * [Contact](#contact)
 
 ## About The Project
-The SYDE class profile, was created to give an insight to the SYDE 2025 cohort's 1A term, similar to reports created
-by previous cohorts. We wanted to go a step above and create a website that would document not just our 1A term, but future terms as well.
+The SYDE class profile, was created to give an insight to the SYDE 2025 cohort's 2A term, similar to reports created
+by previous cohorts. We wanted to go a step above and create a website that would document not just our 2A term, but future terms as well.
```

Comment:
I would suggest adding a sentence or two on the purpose of this project.

Something along the lines of "We wanted to discover and document the impact of COVID-19 on the students of our cohort"